createDataset.py

```diff
@@ -4,18 +4,144 @@
 import re
 from datetime import datetime
 
-personName = raw_input('Enter your full name: ')
 fbData = raw_input('Do you have Facebook data to parse through (y/n)?')
 googleData = raw_input('Do you have Google Hangouts data to parse through (y/n)?')
 linkedInData = raw_input('Do you have LinkedIn data to parse through (y/n)?')
 whatsAppData = raw_input('Do you have whatsAppData to parse through (y/n)?')
+discordData = raw_input('Do you have discordData to parse through (y/n)?')
 
 def getWhatsAppData():
-        df = pd.read_csv('whatsapp_chats.csv')
-        responseDictionary = dict()
-        receivedMessages = df[df['From'] != personName]
-        sentMessages = df[df['From'] == personName]
+    personName = raw_input('Enter your full WhatsApp name: ')
+    df = pd.read_csv('whatsapp_chats.csv')
+    responseDictionary = dict()
+    receivedMessages = df[df['From'] != personName]
+    sentMessages = df[df['From'] == personName]
+    combined = pd.concat([sentMessages, receivedMessages])
+    otherPersonsMessage, myMessage = "",""
+    firstMessage = True
+    for index, row in combined.iterrows():
+        if (row['From'] != personName):
+            if myMessage and otherPersonsMessage:
+                otherPersonsMessage = cleanMessage(otherPersonsMessage)
+                myMessage = cleanMessage(myMessage)
+                responseDictionary[otherPersonsMessage.rstrip()] = myMessage.rstrip()
+                otherPersonsMessage, myMessage = "",""
+            otherPersonsMessage = otherPersonsMessage + str(row['Content']) + " "
+        else:
+            if (firstMessage):
+                firstMessage = False
+                # Don't include if I am the person initiating the convo
+                continue
+            myMessage = myMessage + str(row['Content']) + " "
+    return responseDictionary
+
+def getGoogleHangoutsData():
+    personName = raw_input('Enter your full Hangouts name: ')
+    # Putting all the file names in a list
+    allFiles = []
+    # Edit these file and directory names if you have them saved somewhere else
+    for filename in os.listdir('GoogleTextForm'):
+        if filename.endswith(".txt"): 
+            allFiles.append('GoogleTextForm/' + filename)
+
+    responseDictionary = dict() # The key is the other person's message, and the value is my response
```

Comment:
Nit: Comment spacing looks off here. Might've been there before, but I think cleaning it up would be good.